### DISCUSSION TITLE
【求助】怎么在绳网发表情包啊

### DISCUSSION THREAD

**Discussion by Robin-txdy** [share121/inter-knot#4793] [其他]
求一个详细教程，我研究了半天都没研究明白（PS：我是手机端）拜托了！！！
![Uploading IMG_20231020_160550.png…]()

**Comment by lllliufeng** [comment]
via浏览器可以安装脚本
在绳网上搜表情包就可以找到发表情包的脚本了<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/4b4fb49d4bd402c530600e758ebb06bd_5477506439621383475.png" alt="阿姬开心" width="75" >

**Reply by Robin-txdy** [reply]
为什么我下载脚本显示链接无效啊

**Reply by Robin-txdy** [reply]
大佬方便说说详细的过程吗？

**Comment by HyperMuteki0347** [comment]
在下载脚本的时候记得关梯子（

**Comment by March-777th** [comment]
在via设置`脚本`中勾选`启用脚本`
然后点[这个链接](https://dave-12138.github.io/miyoushe-emoticon-in-discussion/miyoushe-emot-to-github.user.js)
等待自动解析安装

**Reply by Robin-txdy** [reply]
勾选完后，只要点进链接就可以了？

**Reply by March-777th** [reply]
如果没有自动解析，就更新下via

**Reply by Robin-txdy** [reply]
我那个里面显示安装完成了但是还是没有用啊😭

**Comment by Robin-txdy** [comment]
可以用了可以用了，谢谢各位的帮助<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/32fc2d5e0929741455f506c6bd06e066_4709763683651054682.png" alt="米游姬 抛心心" width="75" >
